wiki/Tournaments/TMC/4th/en.md

```diff
@@ -0,0 +1,234 @@
+6. Teams may ask for a rematch if a team member encounters technical difficulties while playing.
+   - "Lag spikes" are not considered a valid reason to nullify a beatmap.
+   - If a rematch happens, the original roster for each team during that particular beatmap must remain the same. If that is not possible, e.g., by virtue of a technical issue, *both* teams will be allowed to swap rosters.
+   - This rule is not to be abused. Referees may veto a rematch request if they find that this is the case.
+7. Disconnects within 30 seconds or 25% of the beatmap length (whichever happens first) can be rematched, at the referee's discretion, as long as it's clearly communicated in the chatroom.
+8. If a player disconnects, their scores will not be counted towards their team's total unless adequate proof of said score is provided. The following are considered acceptable proof:
+   - Livestream clips or VODs from the player's point of view. The entirety of the play and the results screen must be clearly visible along with the affected player's score.
+   - Replay files of the play, taken directly from the "Local scores" tab on the affected player's client (the timestamps must exactly match the time at which the game took place, as seen in the multiplayer match history).
+   - Screenshots showing the affected player's score must be taken directly from within the game.
+     - Screenshots from the results screen must clearly show the affected player's score. This is the preferred method.
+     - Screenshots taken in-game at the time of disconnection may be accepted. Note that this method does not provide a one-to-one representation of that player's score. Using this method is not encouraged, and it may be denied at the referee's discretion if the information provided is not sufficient to determine the player's score.
+     - All screenshots **MUST** be taken using the game itself; that is, they must be hosted on the `https://osu.ppy.sh/` domain (using `Shift` + `F12`). Any other form of screenshot will be denied.
+     - Player scores may be derived from the official stream as a last resort in cases where the match is streamed.
+9. If less than the minimum amount of required players are present at match time, the match can be postponed for up to 10 minutes. If after this period there are still not enough players for either team, a *win by default* will be declared for the side with the most members present.
+   - The minimum number of players required is defined as the number needed to play a beatmap without any empty spots in the lobby (e.g., at least two participants must be present for the match to start).
+   - In the off chance that both teams have the same amount of players (i.e., each team with 1 player present) but not enough to start the match, the match may be rescheduled to a different time; otherwise, a *win by default* will be declared to the team with the highest seed.
```

Comment:
```suggestion
   - In the off chance that both teams have the same amount of players (i.e., each team with 1 player present) but not enough to start the match, the match may be rescheduled to a different time. Otherwise, a *win by default* will be declared to the team with the highest seed.
```